commands/payment_channel_daemon_test.go

```diff
@@ -68,7 +68,7 @@ func TestPaymentChannelLs(t *testing.T) {
 	t.Run("Works with default payer", func(t *testing.T) {
 		t.Parallel()
 
-		payer, err := address.NewFromString(fixtures.TestAddresses[0])
+		payer, err := address.NewFromString(fixtures.TestAddresses[2])
```

Comment:
curious why the renumbering is required here and elsewhere?

Reply:
I have the same question, can't find any other place where [0] gets used

Reply:
TestAddress[0] is the owner of TestMiner[0], so it now gets paid for mining. Most of these tests are concerned with tracking the payer account's balance. We could work around this in the tests, but it's easier just to remove it as a complication.